performance-storage-service/pss_project/api/models/rest/metadata/JenkinsMetadata.py

```diff
@@ -1,3 +1,3 @@
 class JenkinsMetadata(object):
-    def __init__(self, build_id):
-        self.build_id = build_id
\ No newline at end of file
+    def __init__(self, jenkins_job_id):
+        self.jenkins_job_id = jenkins_job_id
```

Comment:
Same comment as above. We probably don't need the `jenkins_`

Reply:
+1

Reply:
> Same comment as above. We probably don't need the `jenkins_`

Same reply as above.